python/neuroglancer/pipeline/scripts/remap2npy.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+
+import sys
+
+import h5py
+import numpy as np
+
+if len(sys.argv) == 1:
```

Comment:
you should also document what the expected remap file looks like. i.e. dimensions, etc.